src/lib.rs

```diff
@@ -16,11 +19,26 @@ pub fn start_server() {
     Server::new();
 }
 
+#[pyfunction]
+pub fn async_static_files(py: Python, file_name: String) -> PyResult<PyObject> {
+    pyo3_asyncio::tokio::into_coroutine(py, async move {
+        let contents = fs::read(file_name.clone()).await.unwrap();
+        let foo = String::from_utf8_lossy(&contents);
+        Ok(Python::with_gil(|py| {
```

Comment:
Surprisingly this approach is coming out to be slower than the python response.
This is probably because of locking and unlocking of gil.